--- a/OVIMGeneric/Makefile.am
+++ b/OVIMGeneric/Makefile.am
@@ -57,7 +57,6 @@
 cj5.cin \
 ckc.cin \
 dapu.cin \
-dayi3.cin \
 dna.cin \
 ez.cin \
 jinjin.cin \
@@ -76,14 +75,10 @@
 dapu.cin \
 hailu.cin \
 kks_hakka_henry.cin \
-kks_hakka_tl.cin \
-sixian.cin \
-tp_hakka_hl.cin \
-tp_hakka_sy.cin
+sixian.cin
 
 CIN_zh_TW_taigi= \
 kks.cin \
-kks_hakka_tl.cin \
 kks_tl.cin \
 phofsi.cin \
 tlpa.cin
--- a/OVIMGeneric/Makefile.in
+++ b/OVIMGeneric/Makefile.in
@@ -250,7 +250,6 @@
 cj5.cin \
 ckc.cin \
 dapu.cin \
-dayi3.cin \
 dna.cin \
 ez.cin \
 jinjin.cin \
@@ -269,14 +268,10 @@
 dapu.cin \
 hailu.cin \
 kks_hakka_henry.cin \
-kks_hakka_tl.cin \
-sixian.cin \
-tp_hakka_hl.cin \
-tp_hakka_sy.cin
+sixian.cin
 
 CIN_zh_TW_taigi = \
 kks.cin \
-kks_hakka_tl.cin \
 kks_tl.cin \
 phofsi.cin \
 tlpa.cin
--- a/SharedData/Makefile.am
+++ b/SharedData/Makefile.am
@@ -2,7 +2,6 @@
 	bpmf-ext.cin \
 	bpmf.cin \
 	cj.cin \
-	dayi3.cin \
 	ehq-symbols.cin \
 	pinyin.cin \
 	punctuations.cin \
--- a/SharedData/Makefile.in
+++ b/SharedData/Makefile.in
@@ -162,7 +162,6 @@
 	bpmf-ext.cin \
 	bpmf.cin \
 	cj.cin \
-	dayi3.cin \
 	ehq-symbols.cin \
 	pinyin.cin \
 	punctuations.cin \
