--- a/DataTables/Makefile.am
+++ b/DataTables/Makefile.am
@@ -47,7 +47,6 @@
 cj5.cin \
 ckc.cin \
 dapu.cin \
-dayi3.cin \
 dna.cin \
 ez.cin \
 jinjin.cin \
@@ -65,9 +64,7 @@
 hailu.cin \
 kks_hakka_henry.cin \
 kks_hakka_tl.cin \
-sixian.cin \
-tp_hakka_hl.cin \
-tp_hakka_sy.cin
+sixian.cin
 
 CIN_zh_TW_taigi= \
 kks.cin \
--- a/DataTables/Makefile.in
+++ b/DataTables/Makefile.in
@@ -235,7 +235,6 @@
 cj5.cin \
 ckc.cin \
 dapu.cin \
-dayi3.cin \
 dna.cin \
 ez.cin \
 jinjin.cin \
@@ -253,9 +252,7 @@
 hailu.cin \
 kks_hakka_henry.cin \
 kks_hakka_tl.cin \
-sixian.cin \
-tp_hakka_hl.cin \
-tp_hakka_sy.cin
+sixian.cin
 
 CIN_zh_TW_taigi = \
 kks.cin \
