--- a/DataTables/Makefile.am
+++ b/DataTables/Makefile.am
@@ -43,7 +43,6 @@
 4corner5.cin \
 array30.cin \
 array40.cin \
-cangjie.cin \
 cj.cin \
 cj5.cin \
 ckc.cin \
@@ -60,11 +59,9 @@
 scj62.cin \
 simplex.cin \
 simplex5.cin \
-uniliu.cin \
 wu.cin 
 
 CIN_zh_TW_hakka= \
-dapu.cin \
 hailu.cin \
 kks_hakka_henry.cin \
 kks_hakka_tl.cin \
@@ -74,7 +71,6 @@
 
 CIN_zh_TW_taigi= \
 kks.cin \
-kks_hakka_tl.cin \
 kks_tl.cin \
 phofsi.cin \
 tlpa.cin
--- a/DataTables/Makefile.in
+++ b/DataTables/Makefile.in
@@ -231,7 +231,6 @@
 4corner5.cin \
 array30.cin \
 array40.cin \
-cangjie.cin \
 cj.cin \
 cj5.cin \
 ckc.cin \
@@ -248,11 +247,9 @@
 scj62.cin \
 simplex.cin \
 simplex5.cin \
-uniliu.cin \
 wu.cin 
 
 CIN_zh_TW_hakka = \
-dapu.cin \
 hailu.cin \
 kks_hakka_henry.cin \
 kks_hakka_tl.cin \
@@ -262,7 +259,6 @@
 
 CIN_zh_TW_taigi = \
 kks.cin \
-kks_hakka_tl.cin \
 kks_tl.cin \
 phofsi.cin \
 tlpa.cin
